emperor/support_files/js/view-controller.js

```diff
@@ -5,62 +5,75 @@ define([
     'slickgrid',
     'chosen'
 ], function($, _, DecompositionView, SlickGrid, Chosen) {
-
   /**
-   * @name EmperorViewControllerABC
-   *
-   * @class Abstract base class for view controllers used in Emperor.
-   * This includes common functionality shared across all of the tab
-   * controllers.
    *
-   * @property {String} [title=""] Title of the controller.
-   * @property {Node} [header=div node] jQuery element for the header
-   * which contains the uppermost elements displayed in a tab.
-   * @property {Node} [body=div node] jQuery element for the body,
-   * which contains the lowermost elements displayed in tab.
-   * This goes below the header.
-   * @property {Node} [gridDiv=div node] jQuery element for the div containing
-   * the slickgrid of sample information.
-   * @property {Node} [canvas=div node] jQuery element for the canvas,
-   * which contains the header and the body.
-   * @property {Node} [container=div node] jQuery element for the parent
-   * container.
-   * This only contains the canvas.
-   * @property {Boolean} [active=false] Indicates whether the tab is front most
-   * @property {String} [identifier="EMPtab-xxxxxxx"] Unique hash identifier
-   * for the tab instance.
-   * @property {Boolean} [enabled=true] Indicates if tab can be accessed.
-   * @property {String} [description=""] Human-readable description of the tab.
-   *
-   */
-
-  /**
-   *
-   * @name EmperorViewControllerABC
+   * @class EmperorViewControllerABC
    *
    * Initializes an abstract tab. This has to be contained in a DOM object and
    * will use the full size of that container.  The title represents the title
    * of the jQuery tab.  The description will be used as help text to describe
    * the functionality of each subclass tab.
    *
-   * @param {Node} [container] Container node to create the controller in.
-   * @param {String} [title] title of the tab.
-   * @param {String} [description] helper description.
+   * @param {Node} container Container node to create the controller in.
+   * @param {String} title title of the tab.
+   * @param {String} description helper description.
    *
    * @return {EmperorViewControllerABC} Returns an instance of the
    * EmperorViewControllerABC.
+   * @constructs EmperorViewControllerABC
    *
    */
   function EmperorViewControllerABC(container, title, description) {
+    /**
+     * @type {Node}
+     * jQuery element for the parent container.
+     */
     this.$container = $(container);
+    /*
```

Comment:
Can you add another `*`, it seems like that's what marks the block as a jsdoc annotation (at least in the syntax highlighter).